### DISCUSSION TITLE
gsoc About application

### DISCUSSION THREAD

**Discussion by 18yz153** [openvinotoolkit/openvino#16744] [Google Summer of Code]
I thought we have to finish the prerequisite task before applying. I think I missed the deadline. However, I finished my prerequisite task. Is that possible to apply? Can I still apply for the project? If I can apply, could you provide me with the portal?

**Comment by adrianboguszewski** [comment]
Did you register here: https://summerofcode.withgoogle.com/?

**Comment by 18yz153** [comment]
How can I tell if I did register or not? I think I did use that website. But I forget if I registered or not.

**Reply by adrianboguszewski** [reply]
Try to log in there using your Google account.

**Reply by 18yz153** [reply]
I didn't. Can I still apply? Or, only contribute to the notebook?

**Reply by adrianboguszewski** [reply]
Unfortunately, you have missed the deadline, then. Applying through the GSoC site is a requirement of Google. However, you can still be part of our community or apply to [Intel Software Innovator Program for Edge](https://www.intel.com/content/www/us/en/developer/community/edge-innovator-program.html)

**Reply by 18yz153** [reply]
It is sad to miss the deadline. Thank you for your help.

**Reply by adrianboguszewski** [reply]
Yes. I hope you can join us in 2024 :)

**Comment by mhy-666** [comment]
Only when you have submitted a PR will you be considered as having completed the prerequisite task.

**Reply by adrianboguszewski** [reply]
That's correct.
